rll/metadata.rb

```diff
@@ -37,9 +37,9 @@
   :default => "env:RS_INSTANCE_UUID",
   :recipes => ["rll::collectd"]
 
-attribute   "DISABLE_AUTO_UPGRADE",
-  :display_name => "Disables auto upgrade if previously enabled",
+attribute   "ENABLE_AUTO_UPGRADE",
+  :display_name => "Enables auto upgrade",
```

Comment:
can you please add " of RightLink10" to make it clear what is being auto-upgraded